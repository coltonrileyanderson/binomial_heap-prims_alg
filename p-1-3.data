6 4 246 ; 6 3 388 ; 2 0 15 ; 3 7 421 ; 0 7 130 ; 
1 7 376 ; 6 5 154 ; 4 3 24 ; 1 2 84 ; 3 3 97 ; 

