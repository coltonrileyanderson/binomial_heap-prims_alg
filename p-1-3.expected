0: 6
1: 4(6)246 5(6)154
2: 3(4)24
3: 7(3)421
4: 0(7)130
5: 2(0)15
6: 1(2)84
weight: 1074
